src/main/java/net/logstash/logback/appender/AbstractLogstashTcpSocketAppender.java

```diff
@@ -80,6 +82,11 @@
     public static final int DEFAULT_RECONNECTION_DELAY = 30000;
 
     /**
+     * The default reconnection to primary server delay (60000 milliseconds or 60 seconds)
+     */
+    public static final int DEFAULT_RECONNECTION_PRIMARY_DELAY = 60000;
```

Comment:
Disabling this feature would mean the appender would stay on the connected host until a failure occurs. In other words, it wouldn't try to automatically reconnect on the primary. Some people may want that behaviour indeed...